ui_framework/components/color_picker/color_picker.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ChromePicker } from 'react-color';
+
+export class KuiColorPicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showColorSelector: false,
+    };
+  }
+
+  closeColorSelector = () => {
+    this.setState({ showColorSelector: false });
+  };
+
+  toggleColorSelector = () => {
+    this.setState({ showColorSelector: !this.state.showColorSelector });
+  };
+
+  handleColorSelection = (color) => {
+    this.props.onChange(color.hex);
+  };
+
+  render() {
```

Comment:
It turns out that this doesn't work well when there are multiple color pickers available on the same page, and you click from one immediately to the other.  
<img width="455" alt="screen shot 2017-06-21 at 4 21 54 pm" src="https://user-images.githubusercontent.com/16563603/27404929-c256111c-569d-11e7-9808-49db57bb0b4d.png">

`e.nativeEvent.stopImmediatePropagation();` stops the other color pickers from closing.

Have you run into this caveat at all @zinckiwi?